api_app/analyzers_manager/constants.py

```diff
@@ -1,31 +1,31 @@
 # This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
 # See the file 'LICENSE' for copying permission.
 
-from enum import Enum
+from django.db import models
 import typing
 
 
-class BaseEnum(Enum):
+class BaseChoices(models.TextChoices):
     @classmethod
     def aslist(cls) -> list:
-        return [c.value for c in cls]
+        return [c for c in cls]
 
     @classmethod
     def as_type(cls) -> typing.Literal:
```

Comment:
Unrelated to this PR but I'd still appreciate if you can make this change.

The `as_type` method is raising many LGTM bot alerts. Can you revert back this change ? Define the typings directly in the `dataclasses.py` file as was done before.

Reply:
After this and the `aslist` change, the `BaseChoices` class can be removed as a whole.